cmd/ncproxy/nodenetsvc/nodenetsvc.proto

```diff
@@ -6,36 +6,57 @@ service NodeNetworkService {
     rpc ConfigureNetworking(ConfigureNetworkingRequest) returns (ConfigureNetworkingResponse);
     rpc ConfigureContainerNetworking(ConfigureContainerNetworkingRequest) returns (ConfigureContainerNetworkingResponse);
     rpc PingNodeNetworkService(PingNodeNetworkServiceRequest) returns (PingNodeNetworkServiceResponse);
+    rpc GetHostLocalIpAddress(GetHostLocalIpAddressRequest) returns (GetHostLocalIpAddressResponse);
```

Comment:
This was just new additions from nns, nothing you specifically added for this test agent I'm gathering? (Judging from it having a pretty barebones implementation for the test agent)